.travis.yml

```diff
@@ -50,10 +50,10 @@ jobs:
     - PLATFORM_PUBLIC_PATH=https://marketplace-staging.streamr.com
     - GOOGLE_ANALYTICS_ID=55331565-5
     - STORYBOOK_BASE_PATH=storybook
-    - MARKETPLACE_CONTRACT_ADDRESS=0x438229f341AEa22f18E69D98c2bE5997bc62d578
-    - DATA_TOKEN_CONTRACT_ADDRESS=0x67a9fda28c4df570d1d5350acfa345194f4d4100
+    - MARKETPLACE_CONTRACT_ADDRESS=0xF1371c0f40528406dc4f4cAf89924eA9Da49E866
```

Comment:
Why are these changing?

Reply:
Those are the values in https://docs.google.com/document/d/1tygb8_9Y5AmzHzio3s3lM0rw_APGFT9hnNNutpUjO8c/edit#, I updated them to match. No idea which version is actually correct

Reply:
That makes 2 of us 😅  but lets give it a go I suppose!